src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -433,8 +433,6 @@ abstract class ConnectionBase: ISelectClient
                 // Register the socket for both input and output
                 this.outer.registerEpoll(true);
                 this.outer.sender.finishSending(this.suspend());
-                if (this.flush_requested)
```

Comment:
Should `this.flush_requested` be deprecated as well?

Reply:
Yes, good catch.